docker-compose.yml

```diff
@@ -66,7 +66,7 @@ services:
         condition: service_healthy
       redis:
         condition: service_healthy
-    command: sh -c "/opi-spdk-bridge -grpc_port=50051 -http_port=8082 -spdk_addr=/var/tmp/spdk.sock -tcp_trid=$$(getent hosts spdk | awk '{ print $$1 }'):7777"
+    command: sh -c "/opi-spdk-bridge -grpc_port=50051 -http_port=8082 -spdk_addr=/var/tmp/spdk.sock -tcp_trid=$$(getent hosts spdk | awk '{ print $$1 }'):777"
```

Comment:
ugly hack! should be addressed in https://github.com/opiproject/opi-api/pull/375

Reply:
or we can pass `-tcp_trid="not_secured_endpoint;secured_endpoint"` since at this stage tcp transport in frontend is more for demo purposes and will be removed after https://github.com/opiproject/opi-api/pull/375 is implemented